spec/lib/log_parser_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../../lib/log_parser'
+
+describe 'LogParser' do
+  describe '#initialize' do
+    it 'raise an exception if recive invalid path' do
+      expect { LogParser.new('some invalid path') }.to raise_error('Files does not exist on main directory')
+    end
+  end
+
+  describe '#read_file' do
+    it 'read game.log first line' do
+      game_parser = LogParser.new('./spec/fixtures/game_test.log')
+
+      response = game_parser.read_line
+
+      expect(response).to eq("  0:00 ------------------------------------------------------------\n")
```

Comment:
Don't you think it would be nice if your parse removes the leading empty spaces?

So, your result will be like that: `"0:00 ------------------------------------------------------------\n"`